ReleaseProcessAutomation/Jira/JiraVersionCreator.cs

```diff
@@ -70,4 +70,16 @@ public string CreateNewVersionWithVersionNumber (string versionNumber)
 
     return createdVersionID;
   }
+
+  public JiraProjectVersion? FindVersionWithVersionNumber (string versionNumber)
+  {
+    var versions = _projectVersionFinder.FindVersions(_config.Jira.JiraProjectKey, "(?s).*").ToList();
+     return versions.Where(x => x.name == versionNumber).DefaultIfEmpty().First();
+  }
+  
+  public IEnumerable<JiraProjectVersion> FindAllVersionsWithVersionNumber (string versionNumber)
+  {
+    var versions = _projectVersionFinder.FindVersions(_config.Jira.JiraProjectKey, "(?s).*").ToList();
+    return versions.Where(x => x.name.Contains(versionNumber));
```

Comment:
Contains can have some nasty side-effects. I assume we want to get all the pre-release versions for a certain version. e.g. `3.1.0` -> `3.1.0-alpha.1`, `3.1.0-beta.2`, ... If this assumption is true I think we should use `.StartsWith` instead and change the name to reflect the change (e.g. `FindAllVersionsStartingWith`)